<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModificarPorta</class>
 <widget class="QDialog" name="ModificarPorta">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>280</width>
    <height>360</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modificar Porta</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>-120</x>
     <y>320</y>
     <width>341</width>
     <height>31</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_portaN">
   <property name="geometry">
    <rect>
     <x>157</x>
     <y>20</y>
     <width>120</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>text</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_PORTA">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>121</width>
     <height>33</height>
    </rect>
   </property>
   <property name="text">
    <string>PORTA:</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>231</width>
     <height>261</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="label_tipo">
        <property name="text">
         <string>Tipo da porta:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="tipo_comboBox">
        <property name="editable">
         <bool>false</bool>
        </property>
        <property name="currentText">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="label_entr">
        <property name="text">
         <string>NÂº de entradas:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="entrd_spinBox">
        <property name="wrapping">
         <bool>true</bool>
        </property>
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>4</number>
        </property>
        <property name="value">
         <number>1</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_7">
      <item>
       <widget class="QLabel" name="label_id1">
        <property name="text">
         <string>Id da entrada 1:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="id1_spinBox">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QLabel" name="label_id2">
        <property name="text">
         <string>Id da entrada 2:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="id2_spinBox">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_6">
      <item>
       <widget class="QLabel" name="label_id3">
        <property name="text">
         <string>Id da entrada 3:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="id3_spinBox">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QLabel" name="label_id4">
        <property name="text">
         <string>Id da entrada 4:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="id4_spinBox">
        <property name="wrapping">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ModificarPorta</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ModificarPorta</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
